### ISSUE TITLE
Module compilation error - cause: unable to resolve class JsonSlurper

### ISSUE THREAD

**Issue by vinsilico** [nf-core/spatialvi#20]
### Description of the bug

Hello,

I am getting this Module compilation error, when I tried to run the test profile. I have groovy 4.0.5 installed. (N E X T F L O W  ~  version 22.04.5). Any advice will be much appreciated.

### Command used and terminal output

```console
nextflow spatialtranscriptomics/main.nf -profile test,singularity --input /PATH/TO/samplesheet.csv

Module compilation error
- file : /Users/xxxx/Documents/SpatialTranscriptomics/spatialtranscriptomics/./workflows/../subworkflows/local/../../modules/local/tasks.nf
- cause: unable to resolve class JsonSlurper
 @ line 201, column 19.
       sample_info = new JsonSlurper().parse(new File(fileName))
                     ^

Module compilation error
- file : /Users/xxxxx/Documents/SpatialTranscriptomics/spatialtranscriptomics/./workflows/../subworkflows/local/../../modules/local/tasks.nf
- cause: unable to resolve class JsonSlurper
 @ line 340, column 19.
       sample_info = new JsonSlurper().parse(new File(fileName))
```

### Relevant files

l

### System information

N E X T F L O W  ~  version 22.04.5

tried in both HPC and MacBook Pro (M1)

local

singularity

CentOS or macOS

dev version

**Comment by mhjiang97** [comment]
I encountered this same bug. Still don't know how to fix it by myself

**Comment by fasterius** [comment]
Thanks for the bug report! This is fixed in #17.
